src/agent/src/namespace.rs

```diff
@@ -82,6 +81,10 @@ impl Namespace {
 
         let ns_path = PathBuf::from(&self.persistent_ns_dir);
         let ns_type = self.ns_type;
+        if ns_type == NamespaceType::PID {
+            return Err(anyhow!("Cannot persist namespace of {} type",
```

Comment:
how about
```diff
+ return Err(anyhow!("Cannot persist namespace of PID type"));
- return Err(anyhow!("Cannot persist namespace of {} type",
-                               NamespaceType::PID.get()));
```

Reply:
@liubin I don't have a strong preference. so I will accept your suggestion. I will also fix any style problem that led to the fail on travis CI.